lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -856,3 +856,72 @@ def test_streamplot():
     ax.streamplot(x, y, u, v, transform=ccrs.PlateCarree(),
                   density=(1.5, 2), color=mag, linewidth=2 * mag)
     return fig
+
+
+@pytest.mark.natural_earth
+@pytest.mark.parametrize(
+    'map_projection',
+    [ccrs.InterruptedGoodeHomolosine(), ]
+    )
+@pytest.mark.mpl_image_compare()
+def test_annotate(map_projection):
+    """ test a variety of annotate options on mulitple projections
+
+    Annotate defaults to coords passed as if they're in map projection space.
+    `transform` or `xycoords` & `textcoords` control the marker and text offset
+    through shared or independent projections or coordinates.
+    `transform` is a cartopy kwarg so expects a CRS,
+    `xycoords` and `textcoords` accept CRS or matplotlib args.
+
+    The various annotations below test a variety of the different combinations.
+    """
+
+    fig = plt.figure(figsize=(10, 5))
+    ax = fig.add_subplot(1, 1, 1, projection=map_projection)
```

Comment:
```suggestion
    ax = fig.add_subplot(1, 1, 1, projection=ccrs.InterruptedGoodeHomolosine())
```

Reply:
left the projection as a variable (but not a parameterized one) for the map coord transform further down the test.